README.md

```diff
@@ -8,6 +8,7 @@
 - [pdm-version](https://github.com/abersheeran/pdm-version) - Make `pdm version` like `poetry version`
 - [pdm-bump](https://github.com/carstencodes/pdm-bump) - A PDM plugin that behaves like [bump2version](https://github.com/c4urself/bump2version) relying on PEP440 compliant versions
 - [pdm-audit](https://github.com/carstencodes/pdm-audit) - A PDM plugin that hooks into the installation end executes [pip-audit](https://github.com/pypa/pip-audit) right after installation or manually
+- [pdm-sbom](https://githib.com/carstencodes/pdm-sbom) - A PDM plugin that generates Software Bill of Materials as [SPDX](https://spdx.org), [CycloneDX](https://cyclonedx.org) and [JFrog build info](https://buildinfo.org) from `pdm.lock`.
```

Comment:
There's a typo in the pdm-sbom link address: githib → github

Reply:
fixed